src/plugins/serverProfile/GuildProfileModal.tsx

```diff
@@ -170,7 +170,7 @@ function ServerInfoTab({ guild }: GuildProps) {
     const Fields = {
         "Server Owner": owner ? Owner(guild.id, owner) : "Loading...",
         "Created At": renderTimestamp(SnowflakeUtils.extractTimestamp(guild.id)),
-        "Joined At": renderTimestamp(guild.joinedAt.getTime()),
+        "Joined At": guild.joinedAt ? renderTimestamp(guild.joinedAt.getTime()) : "-", // Lurked guild
```

Comment:
```suggestion
        "Joined At": guild.joinedAt ? renderTimestamp(guild.joinedAt.getTime()) : "-", // Not available in lurked guild
```
every little helps :-)